packages/web-components/src/components/cbp-list/cbp-list.scss

```diff
@@ -0,0 +1,68 @@
+/**
+ * @prop --cbp-list-item-color: var(--cbp-color-text-darkest);
+ * @prop --cbp-list-item-color-dark: var(--cbp-color-text-lightest);
+*/
+
+:root {
+  --cbp-list-item-color: var(--cbp-color-text-darkest);
+  --cbp-list-item-color-dark: var(--cbp-color-text-lightest);
+  --cbp-list-padding: var(--cbp-space-4x);
+  --cbp-list-item-padding: 2px;
```

Comment:
this is --cbp-space-half-x

Reply:
ah, didn't see that as an option. updating to this var now